src/Microsoft.OData.Core/ODataBatchReader.cs

```diff
@@ -758,7 +497,7 @@ private void IncreaseBatchSize()
         /// <summary>
         /// Increases the size of the current change set; throws if the allowed limit is exceeded.
         /// </summary>
-        private void IncreaseChangeSetSize()
+        internal void IncreaseChangeSetSize()

```

Comment:
>IncreaseChangeSetSize [](start = 22, length = 21)

This also seems only to be called for the MultipartMixed format; should be called for JSON as well. Consider making protected.